### ISSUE TITLE
Knowledge Alpha content type appears to be obsolete

### ISSUE THREAD

**Issue by chao-xian** [alphagov/publishing-api#2221]
I found https://content-data.publishing.service.gov.uk/metrics/knowledge-alpha

which says there's a content type of [`knowledge_alpha` ](https://github.com/alphagov/publishing-api/blob/main/content_schemas/formats/knowledge_alpha.jsonnet)

It was added in 2018 https://github.com/alphagov/govuk-content-schemas/blob/main/formats/knowledge_alpha.jsonnet

No page exists and the URL associated with it is a 404. Should we get rid of it?

**Comment by kevindew** [comment]
Yup, we should. I understand it's in a list of dead ones the content format discovery has found which should hopefully find it onto the teams backlog however a PR to remove it would be welcome :)

Looks like it still lingers in the Content Store: https://www.gov.uk/api/content/knowledge-alpha

**Comment by chao-xian** [comment]
So the lingering appears to be actually what it was meant to be - an API endpoint
